package.json

```diff
@@ -492,9 +492,35 @@
           "command"
         ],
         "properties": {
+          "label": {
+            "type": "string",
+            "description": "The name of the task"
+          },
           "command": {
             "type": "string",
-            "description": "CMake command"
+            "description": "CMake build command"
```

Comment:
let's add:
```
"enum": [ "build" ],
```
to this command object so that people know what's valid to set here.  We can add others as we support them (e.g. "install", "test")